src/contracts/interfaces/IAtlasVerification.sol

```diff
@@ -31,6 +31,7 @@ interface IAtlasVerification {
         returns (uint256 result);
 
     function getUserOperationPayload(UserOperation memory userOp) external view returns (bytes32 payload);
+    function getUserOperationHash(UserOperation memory userOp) external view returns (bytes32 payload);
     function getSolverPayload(SolverOperation calldata solverOp) external view returns (bytes32 payload);
     function getDAppOperationPayload(DAppOperation memory dAppOp) external view returns (bytes32 payload);
```

Comment:
See this other audit issue: https://github.com/spearbit-audits/review-fastlane/issues/65

If we can use `calldata` for all of these params instead of `memory` its a little bit cheaper.

Reply:
58eac8f